examples/results/actuator_disk.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+import flow360 as fl
+import flow360.units as u
+from flow360.examples import ActuatorDisk
+
+ActuatorDisk.get_files()
+
+# submit mesh
+volume_mesh = fl.VolumeMesh.from_file(ActuatorDisk.mesh_filename, name="ActuatorDisk-mesh")
+volume_mesh = volume_mesh.submit()
+
+# # submit case using json file
+params = fl.Flow360Params(ActuatorDisk.case_json)
+case = volume_mesh.create_case("ActuatorDisk-example", params)
+case = case.submit()
+
+
+case.wait()
+
+
+results = case.results
+
+# set values needed for units conversion:
+case.params.geometry.mesh_unit = 1 * u.m
+case.params.fluid_properties = fl.air
+
+print(results.actuator_disks.as_dataframe())
```

Comment:
Is data automatically downloaded when as_dataframe is called?

Reply:
yes. All CSV `results.<results_type>.<access_function>` uses lazy loading.